### PR TITLE
#23 :sparkles: allow velocity context values to be passed to group no…

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**
…tificaitons

### CODE REVIEW

File: fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/VelocityNotification.java [hunk 1]
```diff
@@ -76,27 +83,84 @@ public void addToVelocityContext(String velocityContextKey, Object value) {
...
+                }
+            } catch (IOException e) {
+                throw new GenerationException("Error writing group velocity context properties!", e);
+            }
+        }
+    }
+
+    File getGroupVelocityContenctPropertiesFile(File parentFile) {
+        String propertiesFileName = FilenameUtils.getName(String.format("group-%s.properties", group));
+        File parentDirectory = new File(parentFile, NotificationService.NOTIFICATION_DIRECTORY_PATH + group);
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/VelocityNotification.java:134]: False-positive - this is covered by `propertiesFileName` already.

[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/VelocityNotification.java:134]: @sonatype-lift ignore

File: fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/Notification.java [hunk 2]
```diff
@@ -16,10 +16,18 @@ public interface Notification {
 
     /**
      * Returns a grouping that this notification should be displayed in conjunction with.
+     *
      * @return group
      */
     String getGroup();
 
+    /**
+     * Determines is a group is set for this notification.
```
Comment (tylerashcraft) [inline fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/Notification.java]: I assume you meant "Determines if a group..."

**Review by tylerashcraft — APPROVED** [review]
Looks good except for the one comment.

File: fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/Notification.java (hunk 2 above)
Comment (d-ryan-ashcraft) [reply fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/Notification.java]: Good catch - fixed

**Pull request merged**